cli/Cargo.toml

```diff
@@ -62,6 +62,8 @@ semver = "1.0.20"
 # Telemetry
 wukong-telemetry-macro = { path = "../telemetry-macro" }
 wukong-telemetry = { path = "../telemetry" }
+time = { version = "0.3.31", features = ["formatting", "parsing"] }
+anyhow = "1.0.79"
```

Comment:
I'm not sure if we should use `anyhow` for error handling or not, since we already have our own Error Code

Reply:
Yeah, i tried couple implemenations to avoid using anyhow, but this [line](https://github.com/mindvalley/wukong-cli/blob/d5972682b525c45529ba4c2676faa2aa4b16d412/cli/src/auth/google_cloud.rs#L81) forces me to use the anyhow.

I couldn't make it working with our handler, but if you have any idea how to archive this yeah we can remove this as we are doing our own handling

Reply:
ahhh, my bad. I missed this line. Since this is a trait method, then we have to use live with it.